### PR TITLE
Feature/filter

### PR DESCRIPTION
**Pull request by amiryehu**
fix filter component and create login page

### CODE REVIEW

File: src/components/Filter/Filter-style.ts [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import styled from "styled-components";
+import { border } from "@mui/system";
+
+export const boxStyle = {
+    width: "12rem",
+    bgcolor: "white",
+    borderRadius: "14px",
+    // "hover .MuiOutlinedInput-root .MuiInputBase-root .MuiInputBase-colorPrimar": {  // cancel the botder from Box component
```
Comment (ofekbd-moveo) [inline src/components/Filter/Filter-style.ts]: remove comment out

File: src/components/Filter/Filter-style.ts [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+    //     border: "0px", 
+    //   },
+};
+
+export const selectStyle = {
+  borderRadius: "14px",
+  "&:hover .MuiOutlinedInput-notchedOutline": {
```
Comment (ofekbd-moveo) [inline src/components/Filter/Filter-style.ts:12]: What do you mean? hover on selectStyle applies the style on '.MuiOutlinedInput-notchedOutline'? 
Does it work?
If you mean that '.MuiOutlinedInput-notchedOutline' change the style on hover, we generally write like that:
&.MuiOutlinedInput-notchedOutline:hover

File: src/components/Filter/Filter.tsx [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+import { title } from "process";
+
+interface IFilter {
+  title: string;
+  items?: string[];
+  onChange?: () => void;
+}
+
+export default function BasicSelect(props: IFilter) {
+  const [selectedItem, setselectedItem] = React.useState("");
```
Comment (ofekbd-moveo) [inline src/components/Filter/Filter.tsx]: *setSelectedItem

File: src/components/Filter/Filter.tsx [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+  };
+
+  return (
+    <Box sx={{ ...boxStyle }}>
+      <FormControl fullWidth focused={false}>
+        <Select
+          displayEmpty
+          onChange={handleChange}
+          sx={{ ...selectStyle }}
+          renderValue={(value)=> {
```
Comment (ofekbd-moveo) [inline src/components/Filter/Filter.tsx]: Try to save the component's body "clean" from functionality  - move out that part to function

File: src/components/Filter/Filter.tsx [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+import FormControl from "@mui/material/FormControl";
+import Select, { SelectChangeEvent, SelectProps } from "@mui/material/Select";
+import { boxStyle, selectStyle } from "./Filter-style";
+import { InputLabel } from "@mui/material";
+import { ReactComponent as DropDownIcon } from "../../assets/Icons/dropdown.svg";
+import { title } from "process";
+
+interface IFilter {
+  title: string;
+  items?: string[];
```
Comment (ofekbd-moveo) [inline src/components/Filter/Filter.tsx]: Why it's optional?

File: src/components/Filter/Filter.tsx [hunk 6]
```diff
@@ -0,0 +1,54 @@
...
+              return <em>{selectedItem}</em>
+            }
+          }}
+          value={selectedItem}
+          IconComponent={DropDownIcon}
+        >
+          <MenuItem value="">
+            <em>HARD CODE</em>
+          </MenuItem>
+          <MenuItem value={10}>Ten</MenuItem>
```
Comment (ofekbd-moveo) [inline src/components/Filter/Filter.tsx]: Why it's hardcoded? you should render here the items...

File: src/components/Filter/Filter.tsx [hunk 7]
```diff
@@ -0,0 +1,54 @@
...
+          <MenuItem value={20}>Twenty</MenuItem>
+          <MenuItem value={30}>Thirty</MenuItem>
+        </Select>
+      </FormControl>
+    </Box>
+  );
+}
+function useStyles() {
```
Comment (ofekbd-moveo) [inline src/components/Filter/Filter.tsx]: Using for what?

File: src/components/Filter/Filter-style.ts (hunk 2 above)
Comment (amiryehu) [reply src/components/Filter/Filter-style.ts:12]: In this way I override the design that is in .MuiOutlinedInput-notchedOutline,
only in this way did I manage to cancel the border in select component (mui).

File: src/components/Filter/Filter.tsx (hunk 5 above)
Comment (amiryehu) [reply src/components/Filter/Filter.tsx]: I did this to make it work without data, I'll delete it now

File: src/components/Filter/Filter.tsx (hunk 7 above)
Comment (amiryehu) [reply src/components/Filter/Filter.tsx]: mistake.

File: src/components/Filter/Filter.tsx (hunk 4 above)
Comment (amiryehu) [reply src/components/Filter/Filter.tsx]: Did not succeed

**Review by ofekbd-moveo — APPROVED** [review]

**Pull request merged**